lib/core/toolrepos/step.sh

```diff
@@ -37,15 +35,15 @@ function ih::setup::core.toolrepos::test-or-install() {
       return 1
     fi
     ih::log::info "Cloning engineering repo..."
-    git clone git@github.com:ConsultingMD/engineering.git --filter=blob:limit=1m --depth=5 "${GR_HOME}/engineering" || return
```

Comment:
This repo has been truncated and is no longer GB in size, so we can just clone the whole thing.